src/msftoolbox/gdelt/articles.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+import requests
+from urllib.error import HTTPError
+import newspaper
+
+class GDELTExtractor:
+    def __init__(
+        self,
+        sort="HybridRel",
+        limit=50,
+        mode="ArtList"
```

Comment:
I think that changing the mode to many of the other options might break the whole class. Perhaps we should pin it or create an allowed list.